src/blog/preparing-for-the-merge-hear-from-our-truffle-engineers/index.md

```diff
@@ -0,0 +1,78 @@
+
+### ​​What is the Merge and how does it change Ethereum?
+
+**Kevin Weaver, Senior Blockchain Engineer, Truffle:**
+
+“*Since December 2020, the Ethereum network has encompassed two blockchains running in parallel: the original Proof-of-Work chain, and a new “beacon chain” utilizing a Proof-of-Stake consensus mechanism. The Merge will be the moment these blockchains converge, combining the full history and functionality of the PoW chain with the cleaner, more efficient security of the PoS chain. After the Merge, Ethereum will no longer require burning massive amounts of energy to secure the network, thus reducing its total electrical consumption by about 99.95%.* 
+
+*Given the majority of new ETH issued by the network goes toward paying miners’ electrical bills, this drop in consumption allows for a similarly massive drop in the new issuance of ETH. The combination of this reduction and the fee burning mechanism introduced by [EIP 1559](https://consensys.net/blog/quorum/what-is-eip-1559-how-will-it-change-ethereum), the result will push ETH into becoming a deflationary asset. Importantly, this change is not being made for reasons of political or monetary idealism, but because the network simply no longer requires subsidizing its security with excessive monetary issuance.”*
+
+### Is there a set date for the Merge to happen?
+
+**Kevin:**
+
+*“The tentative date for the Merge is sometime during the week of Sept. 19, 2022. However, this date is subject to change based on the outcome of the Goerli testnet merge.”*
+
+### What do Truffle users need to know/do ahead of the Merge?
+
+**David Murdoch, Lead Blockchain Engineer, Truffle:**
+
+*“They probably don't need to do anything unless they run a node and/or validator. Everything will continue to work at the dApp layer as it does now. There are some changes, but 99.99% of use cases won't care or notice.”* 
+
+**Kevin:**
+
+*“Trufflers do not need to take any actions to prepare for the Merge. The switch to PoS will involve changes at the level of the consensus mechanism, which underpins the application layer, but not on the application layer itself.”*
+
+### What changes should users expect from Ganache?
+
+David on possible upgrades to the Ganache local Ethereum simulator:
+
+*“We will need to add a few new block tags. We currently use ‘latest’, ‘earliest’ and ‘pending.’, Soon, we may add ‘safe’ ‘finalized’, and maybe ‘unsafe.’ What those mean exactly is TBD). Maybe a new block header field and the difficulty field's behavior will change. Almost everything else will be internal changes mostly in the EVM layer. Some advanced users may care about some subtle changes to OPCODE behavior, but those users will be few and far between.”*
```

Comment:
```suggestion
*“We will need to add a few new block tags. We currently use `"latest"`, `"earliest"`, and `"pending"`. We may add `"safe"`, `"finalized"`, and probably `"unsafe"` to this list, though we aren't yet sure the what significance it will have outside of forking. The "difficulty" field and OPCODE will change, as well as a few other small field changes to block headers. Almost everything else will be internal changes, most of which will be in the EVM layer. Some advanced users may care about changes to some OPCODEs, but those users will be few and far between.”*
```